schema.json

```diff
@@ -328,7 +328,7 @@
       "properties": {
```

Comment:
Stage is still missing a description: 

![image](https://user-images.githubusercontent.com/37993418/189552896-b5e9ac1f-8289-4e4f-986d-1d00018adc23.png)

Reply:
Added description for `stage` and other subfields that had none.